tests/unittest/_torch/multimodal/test_multimodal_runtime.py

```
@@ -833,5 +1081,388 @@ def mock_encoder(params):
+            input_ids=input_ids,
+            num_mm_tokens=[3, 2],
+            vocab_size=10,
+        )
+        assert start_pos == [2, 7]
+        assert special_pos == []
+        assert spans == [(2, 3), (7, 2)]
+
+    def test_with_mm_token_ids(self):
+        """MM tokens identified by explicit token IDs."""
+        input_ids = torch.tensor([1, 5, 5, 5, 2, 3, 5, 5, 4])
+        start_pos, special_pos, spans = find_mm_token_positions(
```

Ruff (0.15.10)

[warning] Unpacked variable `special_pos` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)